### DISCUSSION TITLE
After HMR all locals are merged

### DISCUSSION THREAD

**Discussion by thewebartisan7** [posthtml/posthtml#386] [Q&A]
I have an issue with HMR and expressions plugin.

I am using webpack and poshtml-loader, and after HMR all locals are merged together and this require me to set each time the locals for override the previous.

Is there a way to avoid this?

**Comment by Scrum** [comment]
> I have an issue with HMR and expressions plugin.
> 
> I am using webpack and poshtml-loader, and after HMR all locals are merged together and this require me to set each time the locals for override the previous.
> 
> Is there a way to avoid this?

Hello, can you give an example, but is it better to give a link to the repository where it is reproduced?

**Reply by thewebartisan7** [reply]
I will try to reproduce in a new repository as the one where I am working it's private.
